ajax/libs/codemirror-spell-checker/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "codemirror-spell-checker",
+  "version": "1.1.2",
+  "filename": "spell-checker.min.js",
+  "description": "Dead-simple spell checking in CodeMirror",
+  "homepage": "http://nextstepwebs.github.io/codemirror-spell-checker/",
+  "keywords": [
+    "codemirror",
+    "code",
+    "mirror",
+    "spell",
+    "checker",
+    "checking",
+    "spellchecker",
+    "spellchecking",
+    "javascript"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/NextStepWebs/codemirror-spell-checker"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Wes Cossick",
+    "url": "http://www.WesCossick.com"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/NextStepWebs/codemirror-spell-checker.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*.!(*aff|*dic)"
+        ]
+      }
```

Comment:
In addition to files under `dist`, we also need source files under `src`. So please add for example:
```
{
  "basePath": "src"
  "files": [
    "+(css|js)/spell-checker*"
  ]
}
```
You can also design your pattern for `files`!